delius-training-test/sub-projects/parent-orgs.tfvars

```diff
@@ -8,8 +8,8 @@ PO_SPG_CONFIGURATION = {
 
   #THERE IS NO C00 in ND prod yet, nor an assigned crc for testing
   PO_POSTUB_NAME = "PO STUB"
-  PO_POSTUB_TLS_COMMON_NAME = "{{ lookup('env','SPG_CRC_FQDN') }}"
-  PO_POSTUB_SIGNING_COMMON_NAME = "signing.{{ lookup('env','SPG_CRC_FQDN') }}"
+  PO_POSTUB_TLS_COMMON_NAME = "localhost"
```

Comment:
this needs to be the full FQDN, ie spgw-crc-ext.training-test.probation.service.justice.gov.uk